course/src/main/java/in/testpress/course/util/DisplayPDF.kt

```diff
@@ -0,0 +1,70 @@
+package `in`.testpress.course.util
+
+import android.content.Context
+import com.github.barteksc.pdfviewer.PDFView
+import com.github.barteksc.pdfviewer.listener.OnErrorListener
+import com.github.barteksc.pdfviewer.listener.OnLoadCompleteListener
+import com.github.barteksc.pdfviewer.listener.OnPageChangeListener
+import com.github.barteksc.pdfviewer.listener.OnPageErrorListener
+import com.github.barteksc.pdfviewer.scroll.DefaultScrollHandle
+
+class DisplayPDF(
+        val context: Context,
+        private val displayPDFListener: DisplayPDFListener
+) : OnPageChangeListener, OnErrorListener, OnLoadCompleteListener, OnPageErrorListener {
+
+    private var pageNumber: Int = 0
+
+    fun showPdfFromFile(pageNumber: Int, password: String, pdfDownloader: PDFDownloader, pdfView: PDFView) {
```

Comment:
pageNumber can have default value as 1, that way it would be optional. Also make password as optional.
Do not pass pdfDownloader. Instead pass filename or directly file. Change method  name also appropriately.